sections/homepage/Collateral.tsx

```diff
@@ -71,204 +61,65 @@ const Collateral = ({ collateral, tvl, strategies }: CollateralProps) => {
             OETH&apos;s on-chain reserves remain liquid and available for
```

Comment:
A quick hack around using `&apos;s` is to just use backticks, e.g, ```{`OETH's on-chain reserves`}```

Reply:
Yup started doing this... will make the change here as well.